UM/Scene/ToolHandle.py

```diff
@@ -25,15 +25,17 @@ class ToolHandle(SceneNode.SceneNode):
     ZAxis = 4
     AllAxis = 5
 
-    DisabledColor = Color(0.5, 0.5, 0.5, 1.0)
```

Comment:
Toolhandle already has a concept of a "display mesh" versus a "selection mesh" since that was needed for the rotation handles. The simplest would be to simply give all tools a separate display and selection mesh. In fact, I believe the only tool that does not already have a separate mesh is the mirror tool, since it did not need larger hit areas.